Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:carbon
+
+WORKDIR /opt/facility-cache
+ADD . /opt/facility-cache
```

Comment:
@Zooloo2014 Could you please change the docker command from ADD to COPY. ADD has more functionality above COPY (unzipping zipped files and downloading urls) - therefore if we don't need that functionality we should use COPY to prevent something unexpected

https://stackoverflow.com/questions/24958140/what-is-the-difference-between-the-copy-and-add-commands-in-a-dockerfile

Reply:
@Zooloo2014 also here, could you please split out the npm install step from copying the project into the image.

Found this article on Docker best practice. I didn't do this at first as it seemed redundant :|

> Here’s another best practice adjustment: Copy your package.json and package-lock.json before you copy your code into the container. Docker will cache installed node_modules as a separate layer, then, if you change your app code and execute the build command, the node_modules will not be installed again if you did not change package.json.